src/app/backend/dashboard.go

```diff
@@ -116,8 +121,17 @@ func main() {
 
 	// Init integrations
 	integrationManager := integration.NewIntegrationManager(clientManager)
-	integrationManager.Metric().ConfigureHeapster(args.Holder.GetHeapsterHost()).
-		EnableWithRetry(integrationapi.HeapsterIntegrationID, time.Duration(args.Holder.GetMetricClientCheckPeriod()))
+
+	switch metricsProvider := args.Holder.GetMetricsProvider(); metricsProvider {
+	case "sidecar":
+		integrationManager.Metric().ConfigureSidecar(args.Holder.GetSidecarHost()).
+			EnableWithRetry(integrationapi.SidecarIntegrationID, time.Duration(args.Holder.GetMetricClientCheckPeriod()))
+	case "heapster":
+		integrationManager.Metric().ConfigureHeapster(args.Holder.GetHeapsterHost()).
+			EnableWithRetry(integrationapi.HeapsterIntegrationID, time.Duration(args.Holder.GetMetricClientCheckPeriod()))
+	default:
+		log.Printf("Invalid or metrics provider selected: %s", metricsProvider)
```

Comment:
I'd do something like `Invalid metrics provider selected ... Defaulting to 'sidecar'`.

Reply:
Good idea. :+1:

Reply:
`Invalid or metrics provider selected` is it correct?